utils.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+)
+
+func GetAPIKey(keyName string) string {
+	// loads values from .env into the system
+	if err := godotenv.Load(); err != nil {
+		log.Fatal(err)
+	}
+
+	// gets API_KEY
+	apiKey, exists := os.LookupEnv(keyName)
+	if exists {
+		return apiKey
+	}
+	return ""
+}
+
+func HandleTransaction(method, hash string) {
+	if method != "POST" && method != "DELETE" {
```

Comment:
just to understand: what's the point of `DELETE` method in this API?

Reply:
To delete the transaction of hash: `hash` from the list of watched transactions

Reply:
oh ok, interesting. I don't get the use-case though

Reply:
Same tbh. At that point I just implemented the function. 
The docs says the usecase is to [DELETE individual transaction hashes that you are currently watching](https://docs.blocknative.com/mempool-tools/webhook-api#remove-transaction-to-watch). 

I tried to POST a transaction to watch first and the call returns a success msg only:
![image](https://github.com/Amenokal-Labs/mev-mantis/assets/128217762/f69c2a6f-825a-42f4-9d6d-bdfc3d454411)